source/change-streams/change-streams.rst

```diff
@@ -137,7 +137,21 @@ If an aggregate command with a ``$changeStream`` stage completes successfully, t
      * MUST NOT err when they encounter a new `operationType`. Unknown `operationType`
      * values may be represented by "unknown" or the literal string value.
      */
-    operationType: "insert" | "update" | "replace" | "delete" | "invalidate" | "drop" | "dropDatabase" | "rename";
+    operationType: "insert" 
+                  | "update" 
+                  | "replace" 
+                  | "delete" 
+                  | "invalidate" 
+                  | "drop" 
+                  | "dropDatabase" 
+                  | "rename" 
+                  | "createIndexes"
+                  | "dropIndexes" 
+                  | "modify"
+                  | "create" 
+                  | "shardCollection" 
+                  | "refineCollectionShardKey" 
+                  | "reshardCollection";
 
```

Comment:
`refineCollectionShardKey` and `reshardCollection` events are behind a (currently unsupported) feature flag.  I included them here for completeness, but notice that there are no spec tests testing for these new events.  If there are no objections, I can file a follow up ticket to add tests for these events once they're available.